Horizon User Guide/05 Orders/Sales Order.md

```diff
@@ -108,6 +108,21 @@ ___
 
 <img src="https://github.com/Fx-Professional-Services/HorizonDocs/blob/staging/Horizon%20User%20Guide/00%20Assets/05_sales_orders_generate_demand.png" width="350" height="300">
 
+14.2. 
+
+a. To run demand generation asynchronously, click **Generate Demand 2**. This lets you work in other modules while the process runs.
+
+ 💡 Tip: To view running asynchronous tasks, go to **Sales Order** > **Order Tab** and click on **Async Tasks**. You’ll see a list of all processes with their names, start times, and current statuses, which helps in monitoring completion and handling any errors.
```

Comment:
@IamIrish12 I think the location of the Async Task button had changed to the Options sub-menu.

Reply:
I'll update this accordingly. Thanks, Jim!